src/components/ArticleBody.jsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import dayjs from "dayjs";
+
+export default function ArticleBody({ article }) {
+  const newArticle = { ...article };
```

Comment:
you don't need to spread this object here
(hopefully!) you won't be mutating this object so you can use the `article` that's passed down